app/settings/admin/template.hbs

```diff
@@ -0,0 +1,84 @@
+<div class="panel-group">
+  <div class="panel panel-default">
+    <div class="panel-heading"><h3>Change Your Password</h3></div>
+    <div class="panel-body">
+      {{#if passwordError}}
+        <div class="alert alert-warning">
+          {{passwordError}}
+        </div>
+      {{/if}}
+      <form action="">
+        <div class="form-group">
+          <label for="password">New password</label>
+          {{input type="password"
+                  class="form-control" placholder="New password"
+                  value=session.currentUser.password name="password"}}
```

Comment:
I might suggest returning `session.currentUser` as the model of this route, then having the form and actions refer to that. Seems more easily refactored.